relay-config/src/config.rs

```diff
@@ -570,15 +569,19 @@ struct Http {
     encoding: HttpEncoding,
 }
 
+impl Http {
+    pub const DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD: f64 = 10.0;
+}
+
 impl Default for Http {
     fn default() -> Self {
         Http {
             timeout: 5,
             connection_timeout: 3,
             max_retry_interval: 60, // 1 minute
             host_header: None,
-            auth_interval: Some(600), // 10 minutes
-            auth_grace_period: 10,
+            auth_interval: Some(600.0), // 10 minutes
+            network_outage_grace_period: Self::DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD,
```

Comment:
Can we shorten this for easier typing?

```suggestion
            network_grace_period: Self::DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD,
```

or

```suggestion
            outage_grace_period: Self::DEFAULT_NETWORK_OUTAGE_GRACE_PERIOD,
```

Reply:
ok, changed to outage_grace_period